client/allocrunner/network_hook.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/hashicorp/nomad/plugins/drivers"
 )
 
+const dockerNetSpecLabelKey = "docker_sandbox_container_id"
```

Comment:
The specific value surprised me, specially since I didn't see where it's set. I'd suggest adding context and referencing the key in drivers/docker/network.go .

Reply:
Good call.